src/core/primitives.cc

```diff
@@ -115,7 +115,7 @@ CL_DOCSTRING("sleep");
 CL_DEFUN T_sp cl__sleep(Real_sp oseconds) {
   // seconds - a non-negative real.
   SYMBOL_EXPORT_SC_(ClPkg, sleep);
-  double dsec = clasp_to_double(oseconds);
+  double dsec = clasp_to_double(gc::As<Number_sp>(oseconds));
```

Comment:
As above oseconds is a Real_sp - that inherits from Number_sp so...
clasp_to_double(oseconds) is correct

Reply:
compiler says:
../../src/core/primitives.cc:118:17: error: call to 'clasp_to_double' is ambiguous
  double dsec = clasp_to_double(oseconds);
                ^~~~~~~~~~~~~~~
../../include/clasp/core/numbers.fwd.h:51:11: note: candidate function
   double clasp_to_double( core::T_sp );
          ^
../../include/clasp/core/numbers.h:1090:23: note: candidate function
  double              clasp_to_double( core::Number_sp );
                      ^
1 error generated.

will use  double dsec = clasp_to_double(gc::As_unsafe<Number_sp>(oseconds));